helpers/postgres.rb

```diff
@@ -60,7 +60,11 @@ def postgres_list
       }
     else
       dataset = dataset.eager(:representative_server, :timeline)
-      @postgres_databases = Serializers::Postgres.serialize(dataset.all, {include_path: true})
+      resources = dataset.all
+        .group_by { |r| r[:parent_id] || r[:id] }
```

Comment:
Just realized it. This would put forks next to read replicas. How about this instead;

```
.group_by { |r| r.read_replica? ? r[:parent_id] : r[:id] }
```